javascript/apps/taiga/src/app/shared/comments/comments.component.ts

```diff
@@ -81,4 +107,37 @@ export class CommentsComponent {
   public trackById(_index: number, comment: UserComment): string {
     return comment.id;
   }
+
+  public highlightComment(id: string | undefined): void {
+    this.highlightedCommentId = id;
+  }
+
+  private events(): void {
```

Comment:
Story related content should be placed separately. If the story detail is getting too big, you can consider creating a parent element called <tg-story-comments> to handle all the features related to story comments and then use <tg-comments>.